traits/src/traits.rs

```diff
@@ -16,14 +16,14 @@ pub mod types;
 pub trait OpenMlsProvider {
     type CryptoProvider: crypto::OpenMlsCrypto;
     type RandProvider: random::OpenMlsRand;
-    type KeyStoreProvider: key_store::OpenMlsKeyStore;
+    type StorageProvider: storage::StorageProvider<{ storage::CURRENT_VERSION }>;
```

Comment:
I think I made this fixed to the current version. But I'm not sure it's what we should do. Opinions?

Reply:
I think we can leave it this way for now. I don't think it really hurts, and we were planning to take it out of that trait anyway.